ignite/templates/ibc/packet.go

```diff
@@ -346,18 +348,58 @@ func protoTxModify(opts *PacketOptions) genny.RunFn {
 	}
 }
 
+// clientCliTxModify does not use AutoCLI here, because it as a better UX as it is.
 func clientCliTxModify(replacer placeholder.Replacer, opts *PacketOptions) genny.RunFn {
 	return func(r *genny.Runner) error {
-		path := filepath.Join(opts.AppPath, "x", opts.ModuleName, "client/cli/tx.go")
-		f, err := r.Disk.Find(path)
+		filePath := filepath.Join(opts.AppPath, "x", opts.ModuleName, "client/cli/tx.go")
+		if _, err := os.Stat(filePath); errors.Is(err, os.ErrNotExist) {
+			content := fmt.Sprintf(`package cli
+
+			import (
+				"fmt"
+				"time"
+			
+				"github.com/spf13/cobra"
+			
+				"github.com/cosmos/cosmos-sdk/client"
+				"%s/x/%s/types"
+			)
+
+			var DefaultRelativePacketTimeoutTimestamp = uint64((time.Duration(10) * time.Minute).Nanoseconds())
+						
+			// GetTxCmd returns the transaction commands for this module
+			func GetTxCmd() *cobra.Command {
+				cmd := &cobra.Command{
+					Use:                        types.ModuleName,
+					Short:                      fmt.Sprintf("%%s transactions subcommands", types.ModuleName),
+					DisableFlagParsing:         true,
+					SuggestionsMinimumDistance: 2,
+					RunE:                       client.ValidateCmd,
+				}
+			
+				// this line is used by starport scaffolding # 1
+			
+				return cmd 
+			}`, opts.ModulePath, opts.ModuleName) // TODO to improve, probably with a template that does not get copied over directly.
+
+			if err := os.MkdirAll(filepath.Dir(filePath), os.ModePerm); err != nil {
```

Comment:
we can use the `xgenny` package inside the CLI and unbox the templates into the folder. We can also scaffold the `tx.go` by default as the current implementation

Reply:
Or put inside the `commands.go` file

Reply:
Actually, we do not want to scaffold tx.go by default, as we are using autocli by default.
Do you mean adding this specifically in the ibc template instead of the global client one?

Reply:
I mean, instead of creating a file into the go code, we can create a `.plush` file and use the [box method](https://github.com/ignite/cli/blob/679da53781a189b897460f3712a10479d32ec124/ignite/templates/message/message.go#L81)

Or we can also add this method as default into the `.../cmd/commands.go.plush`

Reply:
Mmh okay, I'm going to have a look then 👍